metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -421,8 +423,22 @@ private void reindex(ReindexConfig indexState) throws Throwable {
 
         Pair<Long, Long> tempDocumentsCount = getDocumentCounts(indexState.name(), tempIndexName);
         if (!tempDocumentsCount.equals(documentCounts)) {
-          documentCountsLastUpdated = System.currentTimeMillis();
+          long currentTime = System.currentTimeMillis();
+          long timeElapsed = currentTime - documentCountsLastUpdated;
+          long docsIndexed = tempDocumentsCount.getSecond() - previousDocCount;
+
+          // Calculate indexing rate (docs per millisecond)
+          double indexingRate = timeElapsed > 0 ? (double) docsIndexed / timeElapsed : 0;
```

Comment:
It would be useful to get this indexing rate as a metric.

Reply:
Not a blocker for this logging right? I just want to improve the log in this PR. Feel free to send another PR with adding that metric